Cargo.lock

```diff
@@ -2449,7 +2448,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "33d852cb9b869c2a9b3df2f71a3074817f01e1844f839a144f5fcef059a4eb5d"
 dependencies = [
  "libc",
- "windows-sys 0.59.0",
```

Comment:
Downgrading this and some other dependencies seems undesirable

Reply:
Is its `clap`'s own dependencies getting changed which causes this?

Reply:
weird. it just picks a different windows-sys that's also in our dependency tree. they are all within range for clap, so i don't think it's actually a problem?